### DISCUSSION TITLE
Conductor UI does not pick changes done in .json files in conductor resources unless change in .ts file

### DISCUSSION THREAD

**Discussion by sso679** [Netflix/conductor#2563] [General]
when i do changes in this .json files in conductor-resources /workflow.recorder.initiated.regestration.json and triggered the pipeline and when i see in conductor ui ,i dont see the changes affected in the UI but when i do any changes in .ts file ,conductor ui is picking up changes done in .json files.
could anyone help me please.

**Comment by peterlau** [comment]
Could you please clarify the question? It is unclear what '.ts' and '.json' files are you referring to. Thanks
